terraform/implementation/ecs/destroy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Danger zone! Destroy the ECS cluster
+terraform destroy -auto-approve -var-file="$1.tfvars"
```

Comment:
The auto-approve is the one thing that makes me squidgy here. I understand the reason why it's in place, though!